src/utils/DepositUtils.ts

```diff
@@ -129,7 +130,14 @@ export function isMessageEmpty(message = EMPTY_MESSAGE): boolean {
  * @returns True if the deposit was updated, otherwise false.
  */
 export function isDepositSpedUp(deposit: Deposit): boolean {
-  return isDefined(deposit.speedUpSignature) && isDefined(deposit.newRelayerFeePct);
+  if (!isDefined(deposit.speedUpSignature)) {
+    return false;
+  }
+
+  return (
+    (isV2Deposit(deposit) && isDefined(deposit.newRelayerFeePct)) ||
+    (isV3Deposit(deposit) && isDefined(deposit.updatedOutputAmount))
+  );
```

Comment:
Implemented here: https://github.com/across-protocol/sdk-v2/pull/501/commits/981fae04f60f3ae3d3f306efdc23fe9cb5b14240

I had to overrule prettier because it kept wanting to make the code look dumb :(